src/components/Header.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  
+  <link rel="stylesheet" href="../styles/root.css">
+</head>
+<body>
+<!-------------------------Header Page Start------------------------->
+  
+<div class="ComponentMainBoxOne img-responsive ">
+
+  <header class="headerContainerMain header-center Header-TextSize box-shadow ">
+    <div class="headerContainerOne flex-evenly padding-small-left1">
+      <img class="logoImage" src="./Images/Phoenix UI.jpg" alt="">
```

Comment:
You can add  loading="lazy" in image. it will provide better performance.

Reply:
we use Lazy loading where we have 100 images on a page so it will help to improve performance then, btw thank's for the suggestion I'll do that if there is a performance issue because of images